docker/puppetdb/docker-entrypoint.d/30-configure-ssl.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+if [ ! -f "/etc/puppetlabs/puppet/ssl/certs/${HOSTNAME}.pem" ] && [ "$USE_PUPPETSERVER" = true ]; then
+  # if this is our first run, run puppet agent to get certs in place
```

Comment:
[nitpick] I know this isn't related to your changes but this comment is inaccurate since we're using ssl.sh now